v2/binding/transformer/extract_metadata_test.go

```diff
@@ -0,0 +1,124 @@
+package transformer
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/cloudevents/sdk-go/v2/binding/test"
+	"github.com/cloudevents/sdk-go/v2/event"
+	"github.com/stretchr/testify/assert"
+)
+
+func ExampleExtractExtensions() {
+	e := event.New("1.0")
+	e.Context = &event.EventContextV1{}
+	e.SetID("id")
+	e.SetSource("source")
+	e.SetType("type")
+	e.SetExtension("someextension", "somevalue")
+	e.SetExtension("otherextension", "othervalue") // extension will be ignored
+	extractor := ExtractExtensions{"someextension": nil}
+	_, err := binding.ToEvent(context.TODO(), binding.ToMessage(&e), extractor)
+	if err != nil {
+		fmt.Printf("Error running extractor: %v", err)
+	}
+	fmt.Println("extractor:", extractor)
+	//Output:
+	//extractor: map[someextension:somevalue]
+}
+
+func TestExtractExtensions(t *testing.T) {
+	t.Parallel()
```

Comment:
You don't need this

Reply:
Why not enable parallel execution?

Reply:
Yes but why forcing this from the code? It's enabled by the flag `-parallel`

Reply:
from `go help testflag`:
> -parallel n
>       Allow parallel execution of test functions that call t.Parallel.
>       ...

T.Parallel marks a test as being safe for parallel execution. If it's not called the test will run sequentially with other tests in the same package or, in the case of subtests, with other subtests in the same test. The parallel flag only sets the maximum number of parallel test processes.